plugins/loginmgr/accounts/YggTorrent.php

```diff
@@ -4,6 +4,16 @@ class YggTorrentAccount extends commonAccount
 {
     public $url = "https://yggtorrent.to";
 
+    // Check and change URL automatically if necessary
+    public function __construct()
+    {
+        $headers = @get_headers($this->url, 1);
```

Comment:
As result we will have HTTP request to the yggtorrent on the each object construction.
This is not acceptable, because we iterate over all objects in the function commonAccount::getAccount.
I.e. we will have request to the yggtorrent on the *each* call of Snoopy::fetchComplex.

Reply:
Hm... okay so I will add the new URL directly. ;)